src/libraries/System.DirectoryServices.Protocols/src/System.DirectoryServices.Protocols.csproj

```diff
@@ -1,11 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent);netcoreapp3.1-windows;netcoreapp3.1-OSX;netcoreapp3.1-Linux;netcoreapp3.1;netstandard2.0</TargetFrameworks>
     <IsPackable>true</IsPackable>
     <AddNETFrameworkPlaceholderFileToPackage>true</AddNETFrameworkPlaceholderFileToPackage>
     <AddNETFrameworkAssemblyReferenceToPackage>true</AddNETFrameworkAssemblyReferenceToPackage>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
Not sure how it is got this change @joperezr

Reply:
ah, it was the latest commit was it intentional?

Reply:
This change is needed for servicing, the `GeneratePackageOnBuild` is so that the package is actually built, and the `ServicingVersion` is so that the patch version of the package gets this version, so in this case the package will be 6.0.1

Reply:
This will just make it so the package that gets built version will be `6.0.1` as opposed to `6.0.0`

Reply:
Thanks for explanation